### PR TITLE
Add example showing environment variable usage

### PR DESCRIPTION
**Pull request by benengine**

### CODE REVIEW

File: keras/mnist/keras_mnist.py [hunk 1]
```diff
@@ -234,6 +234,11 @@ def main(args):
                       verbose=1)
   eml.annotate(title='Train', comment='Finished training', tags=[str(args.epochs)])
 
+  # Access environment variables from your engine.yaml file in your code
+  if eml.is_engine_runtime():
+    assert os.environ['WIDGET_TYPE'] == 'gizmo'
+    assert os.environ['MY_SECRET_ENV_VAR'] == 'foo'
```
[the remark this answers is not in the document]
Comment (steve-engineml) [reply keras/mnist/keras_mnist.py]: Also, may be worth adding an error message:
`assert os.getenv('WIDGET_TYPE') == 'gizmo', 'WIDGET_TYPE env variable is not properly set.'`

[the remark this answers is not in the document]
Comment (benengine) [reply keras/mnist/keras_mnist.py]: Done

**Comment by benengine** [comment]
Feedback addressed.

File: keras/mnist/keras_mnist.py [hunk 2]
```diff
@@ -170,6 +170,11 @@ def wait_for_files(a, b, num_retries=10, delay_secs=5):
 
 
 def main(args):
+  # Access environment variables from your engine.yaml file in your code
+  if eml.is_engine_runtime():
+    assert os.environ['WIDGET_TYPE'] == 'gizmo'
```
Comment (steve-engineml) [inline keras/mnist/keras_mnist.py]:
```suggestion
    assert os.getenv('WIDGET_TYPE') == 'gizmo', 'WIDGET_TYPE env variable is not properly set.'
```

File: pytorch/mnist/pytorch_mnist.py [hunk 3]
```diff
@@ -255,6 +255,11 @@ def test(model, test_loader, samples_seen, writer):
 
 
 def main(args):
+  # Access environment variables from your engine.yaml file in your code
+  if eml.is_engine_runtime():
+    assert os.environ['WIDGET_TYPE'] == 'gizmo', 'WIDGET_TYPE env variable is not properly set.'
```
Comment (steve-engineml) [inline pytorch/mnist/pytorch_mnist.py]:
```suggestion
    assert os.getenv('WIDGET_TYPE') == 'gizmo', 'WIDGET_TYPE env variable is not properly set.'
```

File: tensorflow/mnist/tensorflow_mnist.py [hunk 4]
```diff
@@ -269,6 +269,11 @@ def wait_for_files(num_retries=10, delay_secs=5):
 
 
 def main(_):
+  # Access environment variables from your engine.yaml file in your code
+  if eml.is_engine_runtime():
+    assert os.environ['WIDGET_TYPE'] == 'gizmo', 'WIDGET_TYPE env variable is not properly set.'
```
Comment (steve-engineml) [inline tensorflow/mnist/tensorflow_mnist.py]:
```suggestion
    assert os.getenv('WIDGET_TYPE') == 'gizmo', 'WIDGET_TYPE env variable is not properly set.'
```

File: tensorflow/mnist/tensorflow_mnist.py [hunk 5]
```diff
@@ -269,6 +269,11 @@ def wait_for_files(num_retries=10, delay_secs=5):
...
+    assert os.environ['MY_SECRET_ENV_VAR'] == 'foo', 'MY_SECRET_ENV_VAR env variable is not properly set.'
```
Comment (steve-engineml) [inline tensorflow/mnist/tensorflow_mnist.py]:
```suggestion
    assert os.getenv('MY_SECRET_ENV_VAR') == 'foo', 'MY_SECRET_ENV_VAR env variable is not properly set.'
```

File: pytorch/mnist/pytorch_mnist.py [hunk 6]
```diff
@@ -255,6 +255,11 @@ def test(model, test_loader, samples_seen, writer):
...
+    assert os.environ['MY_SECRET_ENV_VAR'] == 'foo', 'MY_SECRET_ENV_VAR env variable is not properly set.'
```
Comment (steve-engineml) [inline pytorch/mnist/pytorch_mnist.py]:
```suggestion
    assert os.getenv('MY_SECRET_ENV_VAR') == 'foo', 'MY_SECRET_ENV_VAR env variable is not properly set.'
```

File: keras/mnist/keras_mnist.py [hunk 7]
```diff
@@ -170,6 +170,11 @@ def wait_for_files(a, b, num_retries=10, delay_secs=5):
...
+    assert os.environ['MY_SECRET_ENV_VAR'] == 'foo'
```
Comment (steve-engineml) [inline keras/mnist/keras_mnist.py]:
```suggestion
    assert os.getenv('MY_SECRET_ENV_VAR') == 'foo', 'MY_SECRET_ENV_VAR env variable is not properly set.'
```

**Pull request merged**